ui/v2.5/src/docs/en/Captions.md

```diff
@@ -0,0 +1,7 @@
+# Captions
+
+Stash supports captioning with SRT and VTT files.
+
+In order for Stash to find these files they must exist in the same directory as the matching video file and must have the same base name. For example, a EN file for `video.mp4` must be named `video.en.vtt`. A scan must be run to update scenes with matching captions.
```

Comment:
With the latest changes and the validation of the lang code i would add something extra to make sure that the language code added is the correct one?
For example mention the `https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes` ? 
Can we also add something explaining that video.srt or video.vtt files are also detected (with the language set as uknown)?